testnets/archwaytestnet/assetlist.json

```diff
@@ -53,6 +53,135 @@
         "website": "https://archway.io/",
         "twitter": "https://twitter.com/archwayHQ"
       }
+    },
+    {
+      "description": "Wrapped CONST (Archway Testnet token)",
+      "type_asset": "cw20",
+      "address": "archway1dz6h2smr432gvzruklk3lps7ak2ca7cngdge4vym93wehkkypwpqgwr2q2",
+      "denom_units": [
+        {
+          "denom": "cw20:archway1dz6h2smr432gvzruklk3lps7ak2ca7cngdge4vym93wehkkypwpqgwr2q2",
+          "exponent": 0
+        },
+        {
+          "denom": "wARCH",
+          "exponent": 18
+        }
+      ],
+      "base": "cw20:archway1dz6h2smr432gvzruklk3lps7ak2ca7cngdge4vym93wehkkypwpqgwr2q2",
+      "name": "Wrapped ARCH",
+      "display": "wCONST",
+      "symbol": "wARCH",
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/archway/images/wArch.png",
```

Comment:
Please use image_sync, and remove all the images that were uploaded to the /archwaytestnet/images/ directory